docs/11-circleci/05-executors.mdx

```diff
@@ -0,0 +1,455 @@
+---
+toc_max_heading_level: 4
+---
```

Comment:
```suggestion
```

Reply:
The 4th level doesn't really add clarity for this page in my opinion. It actually becomes harder to find your executor  from the list like this. Recommend leaving it default.